src/main/java/com/corndel/pixmate/exercises/Exercise3.java

```diff
@@ -12,6 +12,15 @@ public class Exercise3 {
    */
   public static int gridLimit(int[][] grid) {
     // TODO
-    return 0;
+    int total = 0;
+
+    for (int [] outerArray:grid) {
```

Comment:
A good understanding of using nested enhanced for loops to iterate through nested data.